Emby.Server.Implementations/LiveTv/EmbyTV/RecordingHelper.cs

```diff
@@ -48,12 +48,19 @@ public static string GetRecordingName(TimerInfo info)
 
                 if (!string.IsNullOrWhiteSpace(info.EpisodeTitle))
                 {
+                    var tmpName = name;
                     if (addHyphen)
                     {
-                        name += " -";
+                        tmpName += " -";
+                    }
+                    // Calculate the length of the resulting filename
+                    var recordingNameLength = tmpName.Length + info.EpisodeTitle.Length;
+                    var combinedBytes = recordingNameLength * sizeof(Char);
```

Comment:
The `sizeof(char)` is always 2, so you could just require length to be < 125

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/sizeof

Reply:
Cheers! Rebased onto 18.0.z and fixed the line you suggested!